currentRangeDetection.py

```diff
@@ -0,0 +1,55 @@
+MAX_LIMIT = 100
+MIN_LIMIT = 0
+
+def getcurrentvalues(CurrentSamples):
+    if IsRangeOk(CurrentSamples):
+        sorted_CurrentSamples = get_sorted_Samples(CurrentSamples)
+        CurrentRanges_list = get_CurrentRangeslist(sorted_CurrentSamples)
+        message = output_to_csv(CurrentRanges_list)
+        print(message)
+    else:
+        return None
+    return message
+
+def IsRangeOk(values):
+    for item in values:
+        if item > MAX_LIMIT:
+            return False
+    return True
+
+def get_sorted_Samples(CurrentSamples):
+    sorted_array = sorted(CurrentSamples, reverse=False)
+    return sorted_array
+
+def get_CurrentRangeslist(sorted_CurrentSamples):
+    interval = 1
+    temp_arr = []

```

Comment:
The temp_arr is meant to hold something. Rename it to reflect its purpose. Idea is to avoid seeing the full implementation to realize its intent.

Reply:
renamed the array as "CurrentRanges_PrepList"